doc/interop-test-descriptions.md

```diff
@@ -341,27 +341,16 @@ Server features:
 
 
 Procedure:
- 1. Client calls StreamingOutputCall with:
-
-    ```
-    {
-      response_compressed: true
-      response_parameters:{
-        size: 31415
-      }
-      response_parameters:{
-        size: 58979
-      }
-    }
-    ```
+ 1. Client calls StreamingOutputCall with `StreamingOutputCallRequest`:
 
     ```
     {
-      response_compressed: false
       response_parameters:{
+        response_compressed: true
```

Comment:
I see the field is named correctly.
For BoolValue  you might  need to say 
compressed {
  value: true
}

(not sure if protobuf provides syntactic sugar for parsing BoolValue as if it was non-nullable)